### PR TITLE
Milestone4

### PR DESCRIPTION
**Pull request by nyatindopatrick**
Add logic folder, calculate and operate methods.

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line consistent-return
```
Comment (Torres-ssf) [inline src/logic/calculate.js]: - [ ] We should avoid disabling Eslint rules.  Instead, we need to make our code adequate with the specific rule. We can learn more about this specific rule [here](https://eslint.org/docs/rules/consistent-return)

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+const calculate = ({ total, next, operation }, buttonName) => {
+  if (buttonName === '+/-') {
+    return { total: total * -1, next: next * -1, operation };
+  }
```
Comment (Torres-ssf) [inline src/logic/calculate.js]: - [ ] We need to improve the calculate function. Here we need to deal with any possible button that it's pressed within the calculator. Like: `AC`, `.`, `=`, and any digits.

- [ ] Here we also need to deal with the operation. Meaning that, if it is time to do an operation, we need to call the operation function to deal with it.

File: src/logic/operate.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import Big from 'big.js';
+
+// eslint-disable-next-line consistent-return
+const operate = (numberOne, numberTwo, operation) => {
+  // eslint-disable-next-line default-case
```
Comment (Torres-ssf) [inline src/logic/operate.js:5]: - [ ] We need to address the Eslint error here. Check this [link](https://eslint.org/docs/rules/default-case)

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @nyatindopatrick :hand: 
Great job on this milestone so far :clap: :+1: 
But we need to make some changes before moving on.

- [ ] Right now most of your README file is the text generated by `create-react-app`. We need to add a more descriptive text to improve your README file and, therefore, make your repository more professional. It's best practice to add information about how to use your application (instructions on how to set up and which script to use to run your app). You can use this template [here](https://github.com/microverseinc/readme-template), create by Microverse to help to build professional README files. 
Screenshot from 2021-03-03 09-28-06

- [ ] Check my comments below.

Happy coding :computer: 
**Sérgio Torres**

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**